eng/generators.targets

```
@@ -13,7 +14,7 @@
     <!-- If the current project is not System.Private.CoreLib, we enable the LibraryImportGenerator source generator
          when the project is a C# source project that:
          - doesn't target the latest TFM or
-         - references everything from the targeting pack (OOB) or
+         - doesn't reference the targeting pack (i.e. when inbox) and
          - references System.Private.CoreLib, or
          - references System.Runtime.InteropServices -->
```

Let's indent these cases to make it clearer:

```suggestion
             - references System.Private.CoreLib, or
             - references System.Runtime.InteropServices -->
```